File: DTO/TransferObject.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DTO
{
    [Serializable]
    public class TransferObject
    {
        public ErrorData Error { get; set; }
        public ResultData Data { get; set; }
        public string Time { get; set; }
    }
}

File: Front/frontApp/src/app/models/transfer-object.ts
import { Time } from "@angular/common";

export class ErrorData {
    Message: string;
}
export class ResultData {
    DataType: number;
    Message: string;
    Values: any[];
    Headers: string[];
}

export class TransferObject {
    Error: ErrorData;
    Data: ResultData;
    Time: string;
}
